shared/vaccineDataStore.ts

```diff
@@ -59,7 +59,7 @@ export const deleteAllVaccineData = async () => {
 export const getAllRawVaccineSites = async () => {
   const files = await fs.readdir(CONSTANTS.rawSiteFolder);
 
-  return await Promise.all(
+  return Promise.all(
```

Comment:
Why did this change?

Reply:
Minor refactor while in the area - since the caller of this function will await the returned promise, we don't need to "double" await it. 

Can remove if you'd prefer?